packages/mobile/src/screens/track-screen/TrackRemixesScreen.tsx

```diff
@@ -81,6 +81,7 @@ export const TrackRemixesScreen = () => {
       <Header text={messages.header} />
       <Lineup
         lineup={lineup}
+        fetchPayload={{ trackId: track?.track_id }}
```

Comment:
ah i see so this is a way to avoid having to write a custom loadMore? smart

Reply:
Yeah web has custom loadMore's everywhere but this is the first case I found that deviates from the pattern so passing the object made sense

Reply:
i think there may also be a case for profile feed too, where we can use the payload instead of custom loadMore

Reply:
Ohhh didn't know that was a thing